examples/model_hosting/notebook/simple_scheduled_calculation/SimpleScheduledCalculation.ipynb

```diff
@@ -350,7 +350,7 @@
    ],
    "source": [
     "df = ScheduleOutput(prediction).get_dataframe(\"mean\")\n",
-    "df.plot(x=\"timestamp\")"
+    "df.plot()"
```

Comment:
Don't want/need `x="timestamp"` anymore?

Reply:
As above, not anymore.